Tests/test_Restriction.py

```diff
@@ -230,7 +230,7 @@ def test_comparisons(self):
         # Comparison of iso- and neoschizomers
         self.assertEqual(Acc65I, Acc65I)
         self.assertNotEqual(Acc65I, KpnI)
-        self.assertFalse(Acc65I == Asp718I)
+        self.assertNotEqual(Acc65I, Asp718I)
         self.assertFalse(Acc65I != Asp718I)
```

Comment:
There `assertFalse` call looks like it can also be updated in this PR. Would it make sense to update it?

Reply:
233 line brought me a mistake.
Seems like **Acc65I == Asp718I**, so **self.assertNotEqual(Acc65I, Asp718I)** is not correct.

234 line contains the opposite statement. **self.assertEqual(Acc65I, Asp718I)**

May I remove 233?

![Screen Shot 2020-05-01 at 10 11 44 AM](https://user-images.githubusercontent.com/49440211/80789288-3ba04000-8b94-11ea-915e-b69ec2578ec6.png)

Reply:
As I understand, **assertFalse(something not equal)**  in 234 line should become to **assertEqual**

But, when I change:
`self.assertNotEqual(Acc65I, Asp718I)` <- 233 line
`self.assertEqual(Acc65I, Asp718I)` <- 234 line

I got an error in the 233 line.

If I comment it out, assertion error happens on the next line 🤔

![Screen Shot 2020-05-01 at 10 11 44 AM](https://user-images.githubusercontent.com/49440211/80790526-bfa7f700-8b97-11ea-9979-c2cdce9c4290.png)

Reply:
I pass local tests only if I remain **test_comparisons** in **test_Restriction.py** the same.

Reply:
I thought test_Restriction.py was done, see discussion on #2833 - this is a strange case where the objects define conflicting comparison methods.